src/store/facilityList/reducers.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  FacilityList,
+  FacilityActionTypes,
+  UPDATE_FACILITYLIST,
+} from './types';
+
+const initialPerson: FacilityList = {
+  maxCap: 0,
+  currCap: 0,
```

Comment:
in my opinion these should initially be set to `undefined`. This way, if these vars are `undefined` we know there was an issue with the fetch.